src/qt/communityfunddisplaypaymentrequest.cpp

```diff
@@ -81,58 +78,45 @@ void CommunityFundDisplayPaymentRequest::refresh()
     // If prequest is accepted, show when it was accepted
     if (prequest.fState == DAOFlags::ACCEPTED)
     {
-        std::string duration_title = "Accepted on: ";
         std::time_t t = static_cast<time_t>(proptime);
         std::stringstream ss;
-        char buf[48];
-        if (strftime(buf, sizeof(buf), "%c %Z", std::gmtime(&t)))
-            ss << buf;
-        ui->labelTitleDuration->setText(QString::fromStdString(duration_title));
-        ui->labelDuration->setText(QString::fromStdString(ss.str().erase(10, 9)));
+        ss << std::put_time(std::gmtime(&t), "%c %Z");
+        ui->labelTitleDuration->setText(tr("Accepted on: "));
+        ui->labelDuration->setText(QString::fromStdString(ss.str()));
     }
 
     // If prequest is pending show voting cycles left
     if (prequest.fState == DAOFlags::NIL)
     {
-        std::string duration_title = "Voting Cycle: ";
-        std::string duration = std::to_string(prequest.nVotingCycle) +  " of " + std::to_string(GetConsensusParameter(Consensus::CONSENSUS_PARAM_PAYMENT_REQUEST_MAX_VOTING_CYCLES));
-        ui->labelTitleDuration->setText(QString::fromStdString(duration_title));
-        ui->labelDuration->setText(QString::fromStdString(duration));
+        ui->labelTitleDuration->setText(tr("Voting Cycle: "));
+        ui->labelDuration->setText(QString::number(prequest.nVotingCycle) + tr("of") + QString::number(GetConsensusParameter(Consensus::CONSENSUS_PARAM_PAYMENT_REQUEST_MAX_VOTING_CYCLES)));
```

Comment:
Is it not missing blank spaces here? I can't find it on the UI to confirm.

<img width="753" alt="image" src="https://user-images.githubusercontent.com/24814046/66659837-7b2d2c00-ec44-11e9-88d5-ec514ab1762d.png">

Reply:
Nice catch, it is missing spaces.